composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "faktiva/prestashop-seo-tk",
+    "name": "faktiva/faktiva_seo_tk",
```

Comment:
@numanWD what way is the "name" used? Are you sure this is required? What are the alternatives (simply using faktiva_seo_tk for example) ?